### ISSUE TITLE
Add the missing hover focus indicator to the items in the landing page

### ISSUE THREAD

**Issue by ctrance** [ucla-oarc-mobile/casa-on-rails#204]
@krozett 1.	The following are missing hover focus indicators:
a.	UCLA App Store logo link
 ![image](https://user-images.githubusercontent.com/1034174/107822072-5b26e480-6d32-11eb-8cea-5c5f4ff6310d.png)
b.	Search button
 ![image](https://user-images.githubusercontent.com/1034174/107822095-637f1f80-6d32-11eb-9360-abd0de2b599a.png)
c.	All app and see more links
 ![image](https://user-images.githubusercontent.com/1034174/107822107-6aa62d80-6d32-11eb-819b-80fbc85b6b29.png)

Accessibility success criterion
* https://www.w3.org/TR/WCAG21/#use-of-color
* https://www.w3.org/TR/WCAG21/#non-text-contrast

**Comment by ctrance** [comment]
Closing old tickets that we will not fix for now.
